model/src/main/java/jetbrains/jetpad/model/id/BaseId.java

```diff
@@ -107,7 +101,29 @@ public String toString() {
     }
   }
 
-  public static String getEncodedId(long id1, long id2) {
-    return id2 == 0 ? Base64URLSafeCoder.encode(id1) : Base64URLSafeCoder.encode(id1) + SEPARATOR + Base64URLSafeCoder.encode(id2);
+  // generate 128 bits (62 ^ 22) of random readable ID
+  private static String nextRandomId() {
```

Comment:
we use the following order of members in a class:
* static fields
* static methods
* fields
* methods